explorer/src/scss/_solana.scss

```diff
@@ -353,3 +353,10 @@ pre.data-wrap, pre.json-wrap {
 pre.json-wrap {
   max-width: 36rem;
 }
+
+td {
```

Comment:
We should avoid having a global rule for table columns. See the snippet I posted.